_posts/2024-08-02-community-news-august-2024.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: single
 title: "pyOpenSci Community News: August 2024"
-excerpt: "Welcome to the first edition of pyOpenSci's Community News newsletter, where we summarize and share conversations, contributions, and news related to our diverse and vibrant community! Read on for insights into running pyOpenSci sprints, Python packaging guide Spanish translation efforts, and using Hatch to create and maintain Pythons."
+excerpt: "Welcome to the first edition of pyOpenSci's Community News newsletter, where we summarize and share conversations, contributions, and news related to our diverse and vibrant community! Read on for insights into running pyOpenSci sprints, Python packaging guide Spanish translation efforts, and using Hatch to create and maintain Python."
```

Comment:
That would be a much more interesting workshop if we were teaching how to maintain Pythons 🐍